content/docs/getting-started/faq.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Frequently Asked Questions
+---
+
+## General
+
+### Why do you say "Tina is not a CMS?"
+
+Tina is a suite of packages that enable developers to build highly customized content management systems. A conventional CMS provides a fully-realized editing interface, requiring you to create an edit content within its bounds. Tina takes an "inside-out" approach, helping developers craft their own CMS according to their specific content needs.
```

Comment:
```suggestion
Tina is a suite of packages that enable developers to build highly customized content management systems. A conventional CMS provides a fully-realized editing interface, requiring you to create and edit content within its bounds. Tina takes an "inside-out" approach, helping developers craft their own CMS according to their specific content needs.
```

Reply:
Should we say something about conventional CMSs often being connected with a backend as well? 

> a fully-realized editing interface (and often a database or backend)

or something...

Reply:
Yeah, I was thinking along those lines as well. Maybe we need to elaborate a little more